src/lib/staged_ledger/test/main.ml

```diff
@@ -0,0 +1,11 @@
+open Alcotest
+
+let () =
+  run "Test staged ledger."
+    [ Txn_application_test.
+        ( "txn application"
+        , [ test_case
```

Comment:
The tests are not sufficient. From the RFC we need to test the following
1) no transaction depending on a skipped zkApp command is ever applied.
2) if there's less than limit of zkApp commands in the mempool, more signed commands can be applied instead.
3) [not in the RFC] Valid signed commands should be included if `total_space > zkapp_cmd_limit`

I think 2 and 3 are important tests